t3/common.py

```diff
@@ -158,3 +159,33 @@ def convert_termination_time_to_seconds(termination_time: Union[float, str]):
     t_final, units = termination_time
     t_final = t_final*unit_conversion[units]
     return t_final
+
+
+def run_ck2cti(chemkin_file: str,
```

Comment:
RMG _should_ be calling this after model termination. I agree it's a good idea to also allow T3 to have this functionality. Where is this being currently used?

Reply:
RMG does create a cantera file so this helper function is not used by T3. I had added this in case someone wants to use the `CanteraSimulator` for a mechanism not generated by RMG, in which case they may have to convert a chemkin file to a cantera file. Would you prefer I remove this and the corresponding unit test since it's not explicitly used by T3?

Reply:
I see. I think the idea is that T3 will generate a model instead of someone feeding it a Chemkin file. If we need ck2cti we could perhaps use [RMG's functionality](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/3c13a22bc02472f8ea577f4bae69d2c381678570/rmgpy/tools/canteramodel.py#L290). I think we shouldn't include it in T3, but I'm not sure. What do you think?

Reply:
That makes sense. I'll drop the 2 commits that add the function and corresponding unit test.